lib/ConversationContext/styles.scss

```diff
@@ -17,7 +17,7 @@
       border-left: 2px solid $color-border-base;
       position: absolute;
       top: 100%;
-      height: $layout-spacing-base*2;
+      height: 17px;
```

Comment:
This is because the new conversations are a bit further away, so I increased the size of the line that connects them. I didn't want to fall down a hole so I just changed the value, maybe it's something else we could refactor later!?

Reply:
If possible we should fix this soon but I'm down for letting this pass as I assume you only just found this etc.